docs/aspell.en.pws

```diff
@@ -118,3 +118,14 @@ OSX
 githubTipsAndTricks
 ing
 sda
+RVI
+Swupd
+SWUpdate
+Sysup
+TLS
+Uptane
+VIN
+API
+functionalities
+tagname
+SHA
```

Comment:
Not actually related to this specific change, but would it be possible to either force sorting or sort this file manually? Had this thought when I saw another commit where the order was changed automatically.

Reply:
Yep, I'm thinking about doing that actually.